python/lsst/sims/maf/batches/altazBatch.py

```diff
@@ -0,0 +1,132 @@
+import lsst.sims.maf.metrics as metrics
+import lsst.sims.maf.slicers as slicers
+import lsst.sims.maf.metricBundles as mb
+import lsst.sims.maf.plots as plots
+from .colMapDict import ColMapDict
+from .common import standardSummary, extendedMetrics, filterList
+
+__all__ = ['altazHealBatch','altazLambBatch']
+
+def basicSetup(metricName, colmap=None, nside=64):
+
+    if colmap is None:
+        colmap = ColMapDict('opsimV4')
+
+    slicer = slicers.HealpixSlicer(nside=nside, latCol=colmap['alt'], lonCol=colmap['az'],
+                                   latLonDeg=colmap['raDecDeg'], useCache=False)
+    metric = metrics.CountMetric(colmap['mjd'], metricName=metricName)
+
+    return colmap, slicer, metric
+
+def altazHealBatch(colmap=None, runName='opsim', groupName=None,
+                   extraSql=None, extraMetadata=None, metricName='NVisits Alt/Az'):
+
+    """Generate a set of metrics measuring the number visits as a function of alt/az
+    plotted on a HealpixSkyMap.
+
+    Parameters
+    ----------
+    colmap : dict, opt
+        A dictionary with a mapping of column names. Default will use OpsimV4 column names.
+    runName : str, opt
+        The name of the simulated survey. Default is "opsim".
+    groupName : str, opt
+        The group name for this quantity in the displayDict. Default is the same as 'valueName', capitalized.
```

Comment:
Docstring might need some tweaking - 'valueName' is not an argument here.

Reply:
I removed the `groupName` kwarg from the method and docstring since it is not used.